File: 저항/저항.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stout,
});

const colors = [
  'black',
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
];
const colorArr = [];

rl.on('line', line => {
  colorArr.push(line);
  if (colorArr.length === 3) rl.close();
}).on('close', () => {
  console.log(calculateResistorValues(colorArr));
  process.exit();
});

const calculateResistorValues = arr =>
  +(
    (colors.indexOf(arr[0]) * 10 + colors.indexOf(arr[1])) *
    10 ** colors.indexOf(arr[2])
  );

File: 저항/저항.kt
import kotlin.math.pow

fun main() {
    val br = System.`in`.bufferedReader()
    val arr = arrayOf("black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white")
    var str = ""
    repeat(2) {
        val value = arr.indexOf(br.readLine()!!)
        str += value.toString();
    }

    var ans = str.toLong()
    val value = arr.indexOf(br.readLine()!!)
    ans *= 10.0.pow(value).toLong()

    println(ans)
}